Makefile

```diff
@@ -23,7 +23,7 @@ manpage:
 	@cd doc; $(MAKE) manpage
 
 fact-docs:
-	PYTHONPATH=rho:$$PYTHONPATH doc/generate_fact_list.py > doc/facts.rst
+	$(PYTHON) doc/generate_fact_list.py > doc/facts.rst
```

Comment:
This doesn't work for me. Might be an issue in my environment, but I added the environment variable stuff because generate_fact_list.py needs to load the `rho.facts` module.

Reply:
Yours didn't work for me where as the above does. I have no `PYTHONPATH` defined anywhere in my Makefile.

Reply:
Weird! I'm happy to go with your change then, and I know how to make it work for me. I don't think debugging Python library paths is a good use of our time.